charts/brigade-project/values.yaml

```diff
@@ -54,11 +54,12 @@ github:
 ## This is the PRIVATE key that Brigade will use to clone a private repo.
 ## You should generate a special key for this. Don't reuse another one.
 # sshKey: |-
-  # -----BEGIN RSA PRIVATE KEY-----
-  # IIEpAIBAAKCAg1wyZD164xNLrANjRrcsbieLwHJ6fKD3LC19E...
-  # ...
-  # ...
-  # -----END RSA PRIVATE KEY-----
+  #   -----BEGIN RSA PRIVATE KEY-----
+  #   IIEpAIBAAKCAg1wyZD164xNLrANjRrcsbieLwHJ6fKD3LC19E...
+  #   ...
+  #   ...
+  #   -----END RSA PRIVATE KEY-----
```

Comment:
Why the extra indenting added here?

Reply:
Was an indentation mistake, resolved it in my latest commit.